_posts/2024-01-17-January.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: post
+title: "Top of the Year in Minetest (16)"
+contributors:
+  - Zughy
+  - rubenwardy
+  - jordan4ibanez
+  - Bituvo
+  - Apercy
+  - BuckarooBanzay
+  - jdiego
+authors:
+  - MisterE
+description: >
+  Desc Here
+tags:
+  - last_month
+image: /static/blog/2024_TopOfYear/header.jpg
+
+# forum_topic: https://forum.minetest.net/viewtopic.php?f=18&t=29889
+
+# pulls: https://github.com/minetest/minetest/pull
+gallery_apercy:
+- src: /static/blog/2024_TopOfYear/vwbug.png
+  caption: Lovebug 🪲
+- src: /static/blog/2024_TopOfYear/savoia.jpg
+  caption: Savoia S21 from "Porco Rosso"
+- src: /static/blog/2024_TopOfYear/ww1plane.jpg
+  caption: WW1 planes
+gallery_BlEx:
+- src: /static/blog/2024_TopOfYear/better_details.png
+  caption: Better Details page
+- src: /static/blog/2024_TopOfYear/featuredschems.png
+  caption: Featured Schematics
+- src: /static/blog/2024_TopOfYear/login_options.png
+  caption: More Login Options
+gallery_temple:
+- src: /static/blog/2024_TopOfYear/Mortuary_Temple_of_Ramesses_III_at_Medinet_Habu_1.png
+  caption: Overhead view
+- src: /static/blog/2024_TopOfYear/Mortuary_Temple_of_Ramesses_III_at_Medinet_Habu_2.png
+  caption: Inside
+- src: /static/blog/2024_TopOfYear/Mortuary_Temple_of_Ramesses_III_at_Medinet_Habu_3.png
+  caption: The Wall
+---
+
+Desc Here
+
+<!-- more -->
+
+- [Engine News](#engine-news)
+- [Games News](#games-news)
+- [Mods News](#mods-news)
+- [Art and Builds](#art-and-builds)
+- [In Other News](#in-other-news)
+
+## Engine News
+
+### New core dev: Lars
+
+The community welcomes Lars Müller (Luatic) ([Github](https://github.com/appgurueu), [ContentDB](https://content.minetest.net/users/LMD/)), a longtime community
+member, as the newest Core Dev. He is active in the effort to modernize
+Minetest's mesh loader with the GLTF format, and has a PR to limit the observers
+of objects, allowing entities to be hidden from players.
+
+### Gamepad support and Dynamic tablet support
+
+Progress is being made towards moving to SDL2 input support, which will allow
+Minetest to be controlled with a gamepad, mouse and keyboard, or a touchscreen
+on any platform. Gamepad support, and android support of mouse and keyboard
+input, have been two major complaints over the years.
+
+Additionally, work is being done to make it possible to switch between
+touchscreen and mouse mode at runtime, which will allow tablets with
+touchscreens or android devices to switch between modes at will, while playing.
```

Comment:
"tablets with touchscreens"? I guess you mean "laptops with touchscreens"?

you could remove "at will", it's kinda obvious

"android" should be "Android"